cmd/nvidia-dra-controller/imex.go

```diff
@@ -99,29 +99,37 @@ func StartIMEXManager(ctx context.Context, config *Config) (*ImexManager, error)
 
 // manageResourceSlices reacts to added and removed IMEX domains and triggers the creation / removal of resource slices accordingly.
 func (m *ImexManager) manageResourceSlices(ctx context.Context, owner resourceslice.Owner, addedDomainsCh <-chan string, removedDomainsCh <-chan string) error {
-	driverResources := resourceslice.DriverResources{}
+	driverResources := resourceslice.DriverResources{
+		Pools: make(map[string]resourceslice.Pool),
```

Comment:
This shouldn't be necessary. We assign Pool to the map made in the body of the call to `generateImexChannelPool()` in all cases.

Reply:
done